README.md

```diff
@@ -1,13 +1,13 @@
-# Ethereum Protocol Fellowship Study Group
+# Ethereum Protocol Fellowship Study Group (EPFsg)
 
-This repository is dedicated to coordination and knowledge base for EPFsg. It hosts the Protocol Wiki, a community effort to build a technical knowledge base about the Ethereum core protocol. 
+This repository is dedicated to fostering coordination and developing a knowledge base for the EPFsg. It hosts the Protocol Wiki, a community effort to build a technical knowledge base about the Ethereum core protocol. 
 
-Learn more about the EPFsg in announcement blog post and join the study group: https://blog.ethereum.org/2024/02/07/epf-study-group
+Learn more about the EPFsg in the [announcement blog post](https://blog.ethereum.org/2024/02/07/epf-study-group).
 
 ## Protocol Studies Wiki
 
 You can visit the wiki at https://epf.wiki or head to `/docs` for the source. 
 
-The wiki is mostly build by contributors from the EPF Study Group. Join us to learn about the protocol and start [contributing](/docs/contributing.md). 
+The wiki is maintained by students of the EPFsg. [Join us](https://discord.com/invite/addwpQbhpq) to learn about the protocol and start [contributing](/docs/contributing.md). 

```

Comment:
```suggestion
The wiki is maintained by students of the EPFsg. [Join us](https://discord.com/invite/addwpQbhpq) to learn about the protocol and start [contributing](/docs/contributing.md). 

```
The wiki is not exclusively maintained by students and is op en to public contribution

Reply:
I ended up removing that sentence entirely because we state that 

> the Protocol Wiki, [is] a community effort to build a technical knowledge base focused on the Ethereum core protocol.

Because we already explain that the wiki is a community effort it's implied that contributions are open to the public so I don't think it's necessary to explicitly re-state that. Plus we go on to explain this even further in `docs/contributing.md`